_episodes/03-create.md

```diff
@@ -132,12 +132,12 @@ nothing to commit (create/copy files and use "git add" to track)
 > > {: .bash}
 > >
 > > ~~~
-> > moons
+> > /csv
 > > ~~~
 > > {: .output}
 > >
-> > To recover from this little mistake, Dracula can just remove the `.git`
-> > folder in the moons subdirectory. To do so he can run the following command from inside the 'moons' directory:
+> > To recover from this little mistake, you can just remove the `.git`
```

Comment:
This is so weird. Not what you did, but the whole "we made a *little* mistake, let's delete the `.git` directory". Talk about a big hammer..

Reply:
Seriously.